aws/resource_aws_ses_template.go

```diff
@@ -0,0 +1,156 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesTemplate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesTemplateCreate,
+		Read:   resourceAwsSesTemplateRead,
+		Update: resourceAwsSesTemplateUpdate,
+		Delete: resourceAwsSesTemplateDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
This field is unnecessary, every resource gets one automatically. In fact if you rebased to the latest master you'd see a failing test because of that as we [recently](https://github.com/terraform-providers/terraform-provider-aws/pull/2066) improved the schema validation.